### ISSUE TITLE
Add ability to remap input to loading a specific ROM

### ISSUE THREAD

**Issue by richarddb** [stella-emu/stella#981]
To help others experience the 2600 easily with a custom-built macro pad, I’d like to be able to load a specific ROM from a single keypress. Something like a Combo shortcut, which halts the current emulation, selects a ROM, then resets into that ROM. I could have a few keys custom-printed with iconography for the historical most popular games with a single press.

**Comment by thrust26** [comment]
Seems complicated, definitely doesn't fit to our current framework.

**Comment by thrust26** [comment]
@sa666666 Do you have any idea how to handle this?

**Comment by sa666666** [comment]
Yes, and I've assigned myself to this one.  Will get to it before the next release.

**Comment by thrust26** [comment]
Cool. Let me know if I can help e.g. with the UX.

**Comment by thrust26** [comment]
I just had an idea how to integrate this into the UI:

We could use the virtual folders, either a special folder (e.g. named "Playlist) or the "Favorites" folder(s). The direct access hotkeys link to the ROMs in the folder/subfolders (the folder depends on from where the first ROM got started). We should allow manual sorting of the games inside the folder(s). Also "reload" could act like it does for multicarts and switch to the previous/next game in the current folder.

@sa666666 What is your idea here?
